Documentation/Signoffs/Camera_Power_Signoff.md

```diff
@@ -0,0 +1,131 @@
+# Primary Data Acquisition Power Detail Design
+
+## Big Picture
+
+The primary data acquisition subsystem will utilize a number of different electrical components in order to ensure the cameras can transfer images over to the server. Given its high elevation on the outdoor posts, there will be no simple way to wire the subsystem to a grounded power source. Therefore, a “wireless” solution will have to be developed in order to ensure that the subsystem can have a reliable and hands-off way to remain powered, and this design goes over the implementation.
+
+## Specifications
+
+Hardware Specifications
+
+1. Avigilon Cameras [1]
+    * Input Voltage - 12 VDC +/- 10%
+    * Input Power - 6 W (min) 
+    * Power Supply Connections - Power over Ethernet (IEEE 802.3af Class 3), 2-pin terminal block
+2. WiFi Module [2]
+    * Input Voltage - 5-24 VDC
+    * Current Draw - 2 A
+    * Power Supply Connections - USB, barrel plug, DC connector
+3. Arduino Nano 33 IoT [3]
+    * Input Voltage - 3.3-21 V 
+    * Current Draw - 85 mA [4]
+    * Power Supply Connections - USB, Vin & GND pin
+4. Heater Plate [5]
+    * Input Voltage - 5 V
+    * Input Power - 1 W
+    * Power Supply Connection - Positive and Negative wire
+
+Electrical Constraints
+
+1. The power supply must meet the voltage/current needs for all devices requiring power
+2. Devices drawing current from the power supply must be protected from overcurrent
+3. Devices receiving power must use the correct adapters to connect to the power supply
+4. The power supply must be rechargeable to minimize maintenance
+
+Reliability Constraints
+
+1. The power supply must not take up much space
+
+## Buildable Schematic
+
+![new_camera_connections](https://user-images.githubusercontent.com/80428236/223566903-4d639d9c-5294-49fc-8227-97c6a2726850.png)
+
+Figure 1. Electrical Wiring Schematic
+
+## Analysis
```

Comment:
These don't seem to address the constraints. Further, much of this has nothing to do with analyzing the design. 

This should show how you know that the constraints are met. There's no need for paragraphs. Get rid of them. This should be circuit analysis (nodal analysis, spice simulation, etc.)